source/NVDAHelper.py

```diff
@@ -1,3 +1,9 @@
+#NVDAHelper.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2017-2018 NV Access Limited, Peter Vagner, Davy Kager

```

Comment:
Thanks for adding the header, but I don't think 2017 is a very accurate guess here. Could you do a quick search with git blame and change this accordingly?

Reply:
I already know it's 2008, not sure why that got lost.